src/plugins/ReactIntegrationUsageDataPlugin.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import React from 'react';
+import { createIntegrationUsageDataPlugin } from '@ckeditor/ckeditor5-integrations-common';
```

Comment:
Is this import ok? It seems that this plugin is absent. Or are we waiting for some merge in `integrations-common`?

Reply:
Old cache. Nevermind.